controllers/component_controller.go

```diff
@@ -230,7 +230,7 @@ func (r *ComponentReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 
 				// Generate and push the gitops resources if spec.containerImage is set
 				if component.Spec.Build.ContainerImage != "" {
```

Comment:
Is it possible to have `component.Spec.Build.ContainerImage` empty? Seems it's set to default value [here](https://github.com/redhat-appstudio/application-service/blob/46684055dc75b2e05899d8d38cf209de17460ace/controllers/component_controller.go#L97).

Reply:
yeah true, it does seem the default is set there. It was set via a different PR so this check was not removed.

Reply:
Obsolete conditions removed.